include/exiv2/heifimage.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef HEIFIMAGE_HPP
```

Comment:
Two comments here:
1. You would need to include the license template header that you can find in other files.
2. Please, use `#pragma once` instead of header guards. We recently changed all our files in that sense.